spec/diagram.svg

```diff
@@ -11,14 +10,4 @@ database db
 db -> app
 app -> client
 @endum
-
-PlantUML version 1.2020.05(Thu Mar 19 22:17:32 CET 2020)
```

Comment:
I'm not sure it's necessary to trim away this.

Reply:
If I do not the test fails since I remove it from the output

Reply:
But is it necessary to remove at all?

Reply:
It isn't I just did to clean up the output even more.

Reply:
Ok, let's not overdo it. We only need to remove the XML prologue. :)

Reply:
Change has been reverted.